### DISCUSSION TITLE
gh-pages html not rendering?

### DISCUSSION THREAD

**Discussion by oleersoy** [community/community#23000] [Other Feature Feedback, Questions, & Ideas]
<p>I have this gh-pages branch:</p>
<p><a href="https://github.com/superflycss/component-site/blob/gh-pages/deploy/index.html" rel="nofollow noopener">https://github.com/superflycss/component-site/blob/gh-pages/deploy/index.html</a></p>
<p>But index.html is not showing up?</p>
<p><a href="https://superflycss.github.io/component-site/deploy/" rel="nofollow noopener">https://superflycss.github.io/component-site/deploy/</a></p>
<p>Thoughts?</p>

**Comment by Gisgar3** [comment]
<p>I am pretty sure this issue stems from the fact that you’re trying to create a  <strong>GitHub Pages</strong> site via an organization repository that does not fit the requirements for one. According to the  <strong>GitHub Pages</strong> documentation:</p>
<pre><code>Head over to GitHub and create a new repository named username.github.io, where username is your username (or organization name) on GitHub.
</code></pre>
<p>^ this means you would have to deploy a  <strong>GitHub Pages</strong> site in a repository named <strong>superflycss.github.io</strong>; you might also have to fully enable the site in the repository’s settings once it is created.</p>
<p>I hope this helps! If you want to learn more about  <strong>GitHub Pages</strong> and on creating a new site through it, you can visit their <a href="https://pages.github.com/" title="GitHub Pages Documentation" rel="nofollow noopener">documentation</a>.</p>

**Comment by oleersoy** [comment]
<p>I’m using the same process I use for all the other superflycss repositories.  For example this one:</p>
<p><a href="https://www.npmjs.com/package/@superflycss/component-test" rel="nofollow noopener">https://www.npmjs.com/package/@superflycss/component-test</a></p>
<p><a href="https://superflycss.github.io/component-test/target/test/html/" rel="nofollow noopener">https://superflycss.github.io/component-test/target/test/html/</a></p>
<p><a href="https://github.com/superflycss/component-test/tree/gh-pages" rel="nofollow noopener">https://github.com/superflycss/component-test/tree/gh-pages</a></p>
<p>The only difference is that I changed the travis command to publish the html files to deploy instead of target/test/html.  I’m testing serving of optimized CSS files.</p>

**Comment by Gisgar3** [comment, marked as answer]
<p>Ahhh, I see. Not sure if that  <strong>Travis</strong> command you used can somehow differ the process used to display a  <strong>GitHub Pages</strong> site. If possible, I recommend trying your old method again and seeing if that fixes the issue and then move on to brainstorming a new method of attempting to achieve what you want right now.</p>
<p>I personally am not sure of the issue occurring myself, so I am just giving you advice on how to test for a problem. Hope it works out good in the end!</p>

**Comment by oleersoy** [comment]
<p>But your hunch made me look at the settings and it had this in it:</p>
<p>&gt;  Your site is having problems building: The tag import on line 18 in README.md is not a recognized Liquid tag. For more information, see <a href="https://help.github.com/en/articles/page-build-failed-unknown-tag-error" rel="nofollow noopener">https://help.github.com/en/articles/page-build-failed-unknown-tag-error</a>.</p>
<pre><code class="lang-auto">{% import "@superflycss/component-site/SiteHead.njk" as SiteHead %}
{{ SiteHead.head( 'SuperflyCSS Component Site' ) }}
</code></pre>
<p>Weird that it’s complaining about that since it’s part of the README and not the site.  Anyways I’m pretty adding README.md to .gitignore should fix it</p>

**Comment by oleersoy** [comment]
<p>This is what I have for travis now (needed to add <code>- sed -i '1 i README.md' .gitignore</code> so that it ignores README.md when pushing to github pages:</p>
<pre><code class="lang-auto">
language: node\_js
sudo: false
node\_js:
- "stable"
deploy:
provider: pages
skip\_cleanup: true
github\_token: $GITHUB\_TOKEN # Set in travis-ci.org dashboard
on:
branch: master
before\_deploy:
- npm i -g @superflycss/cli
- sfc deploy
- sed -i '1 i README.md' .gitignore
- sed -i '/target/d' .gitignore
- sed -i '/deploy/d' .gitignore

</code></pre>

**Comment by nabilateuse** [comment]
<p>Your site is having problems building: . For more information, see <a href="https://help.github.com/en/articles/page-build-failed-unknown-tag-error" rel="nofollow noopener">https://help.github.com/en/articles/page-build-failed-unknown-tag-error</a>.</p>
<blockquote>
<p><a class="mention" href="/oleersoy">@oleersoy</a> wrote:</p>
</blockquote>
<blockquote>
<p>I have this gh-pages branch:</p>
</blockquote>
<blockquote>
<p><a href="https://github.com/superflycss/component-site/blob/gh-pages/deploy/index.html" rel="nofollow noopener">https://github.com/superflycss/component-site/blob/gh-pages/deploy/index.html</a></p>
</blockquote>
<blockquote>
<p>But index.html is not showing up?</p>
</blockquote>
<blockquote>
<p><a href="https://superflycss.github.io/component-site/deploy/" rel="nofollow noopener">https://superflycss.github.io/component-site/deploy/</a></p>
</blockquote>
<blockquote>
<p>Thoughts?</p>
</blockquote>
